src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -101,7 +108,8 @@ def open_browser(self, url, browser='firefox', alias=None,
         To be able to actually use one of these browsers, you need to have
         a matching Selenium browser driver available. See the
         [https://github.com/robotframework/SeleniumLibrary#browser-drivers|
-        project documentation] for more details.
+        project documentation] for more details. Headless Firefox and
+        Headless Google Chrome require Selenium 3.8.0 or newer.
```

Comment:
Should also mention that they are new in SeleniumLibrary 3.x.x, depending on which version this is going to be added.

Reply:
Lets put this to the 3.1.0 release.